SwiftWisdom/Core/Downloader/Downloader.swift

```diff
@@ -82,3 +74,15 @@ extension URLSessionTask {
         return min(percentage, 1)
     }
 }
+
+// MARK: Operator Overloads
+
+extension Downloader {
+    public static func ==(lhs: Downloader, rhs: Downloader) -> Bool {
```

Comment:
Will you update these extensions (and the others in this PR) to include a space after the operator.